specification/synapse/data-plane/Microsoft.Synapse/preview/2021-07-01-preview/symsSync.json

```diff
@@ -0,0 +1,326 @@
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorContract"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ErrorContract": {
+      "title": "Error details.",
+      "description": "Contains details when the response code indicates an error.",
+      "type": "object",
+      "properties": {
+        "error": {
+          "description": "The error details.",
+          "$ref": "#/definitions/ErrorResponse"
+        }
+      }
+    },
+    "ErrorResponse": {
+      "required": [
+        "code",
+        "message"
+      ],
+      "type": "object",
+      "properties": {
+        "code": {
+          "type": "string"
+        },
+        "message": {
+          "type": "string"
+        },
+        "target": {
+          "type": "string"
+        },
+        "details": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ErrorDetail"
+          }
+        }
+      }
+    },
+    "ErrorDetail": {
+      "required": [
+        "code",
+        "message"
+      ],
+      "type": "object",
+      "properties": {
+        "code": {
+          "type": "string"
+        },
+        "message": {
+          "type": "string"
+        },
+        "target": {
+          "type": "string"
+        }
+      }
+    },
+    "metadataRequestAcceptedResponse": {
+      "type": "object",
+      "properties": {
+        "requestId": {
+          "type": "string"
+        }
+      }
+    },
+    "successResponse": {
+      "type": "object",
+      "properties": {
+        "requestStatus": {
+          "type": "string"
+        }
+      }
+    },
+    "registerObject": {
+      "type": "object",
+      "required": [
+        "inputFolder",
+        "databaseName"
+      ],
+      "properties": {
+        "inputFolder": {
+          "type": "string",
+          "example": "https://testsymsstorage.blob.core.windows.net/testsymsstoragefilesystem/CDM/",
+          "description": "The input folder containing CDM files."
+        },
+        "databaseName": {
+          "type": "string",
+          "example": "testdatabase",
+          "description": "The database name to be created."
+        }
+      }
+    },
+    "updateObject": {
+      "type": "object",
+      "required": [
+        "inputFolder",
+        "databaseName"
+      ],
+      "properties": {
+        "inputFolder": {
+          "type": "string",
+          "example": "https://testsymsstorage.blob.core.windows.net/testsymsstoragefilesystem/CDM/",
+          "description": "The input folder containing CDM files."
+        },
+        "databaseName": {
+          "type": "string",
+          "example": "testdatabase",
+          "description": "The database name to be created."
+        }
+      }
+    },
+    "metadataDeleteObject": {
```

Comment:
>metadataDeleteObject [](http://example.com/codeflow?start=5&length=20)

I see `registerObject` and `updateObject` doesn't have prefix "metadata". Should we add "metadata" prefix for register/updateObject too?

Reply:
added the prefix metastore before all these objects